adapters/handlers/rest/state/state.go

```diff
@@ -43,6 +43,7 @@ type State struct {
 	SchemaManager      *schema.Manager
 	Cluster            *cluster.State
 	RemoteIncoming     *sharding.RemoteIndexIncoming
+	RemoteNodeIncoming *sharding.RemoteNodeIncoming
```

Comment:
Would it be useful to change the above `RemoteIncoming` to `RemoteIndexIncoming` to help differentiate now that we have multiple similar types?